content/_partials/changelog-changes.html.haml

```diff
@@ -35,8 +35,11 @@
                   - else
                     = reference.url
           - elsif change.issue
-            %a{:href => "https://issues.jenkins.io/browse/JENKINS-#{change.issue}" }<>
+            %a{:href => "https://github.com/jenkinsci/jenkins/issues?q=is%3Aissue%20%22jira_issue_key%3DJENKINS-#{change.issue}%22" }<>
```

Comment:
https://github.com/timja-org/jenkins-gh-issues-poc-11-14/issues?q=is%3Aissue%20%22jira_issue_key%3DJENKINS-75990%22